build.sbt

```diff
@@ -35,14 +35,14 @@ val knockoffVersion = "0.8.3"
 lazy val knockoffDeps = Def.setting { Seq(
   "com.tristanhunt" %% "knockoff" % knockoffVersion
 )}
-val unfilteredVersion = "0.8.3"
+val unfilteredVersion = "0.8.4"
 val stringtemplateVersion = "3.2.1"
 lazy val libraryDeps = Def.setting { Seq(
   "net.databinder" %% "unfiltered-filter" % unfilteredVersion,
   "net.databinder" %% "unfiltered-jetty" % unfilteredVersion,
   "org.antlr" % "stringtemplate" % stringtemplateVersion
 )}
-val launcherInterfaceVersion = "0.13.0"
+val launcherInterfaceVersion = "1.0.0"
```

Comment:
Should be `"org.scala-sbt" % "launcher" % launcherInterfaceVersion % "provided"`, probably.